tests/test_celery.py

```diff
@@ -22,6 +23,7 @@ def test_celery_add(celery_app, celery_worker, client):
     assert response.json['result'] == x + y
 
 
+@pytest.mark.skip(reason="locking up with celery fixtures")
```

Comment:
this did work with the celery fixtures, provided (https://github.com/uwcirg/truenth-portal/blob/feature/multi-celery-queues_fixture-job/tests/conftest.py#L54) both queues are in place.

Reply:
This PR is based on yours (I branched from your branch) so it should include that change. I [saw multiple failures](https://gitlab.com/uwcirg/truenth-portal/-/jobs/331143964) on this test and since it wasn't actually a test I skipped it. 

I'm not sure why it would fail now, but maybe we need to reduce the fixture scope after I added more tests to the fixtured test suite?

Reply:
Not clear to me either.  It does require the `low_priority` queue, but so do several other tests that now pass.
I'm okay with keeping the skip in there for now.